Application/Sources/Bridges/SwiftMessagesBridge.swift

```diff
@@ -58,8 +58,12 @@ final class SwiftMessagesBridge: NSObject {
             presentationController = presentationController?.parent
         }
 
-        if let presentationController {
-            config.presentationContext = .viewController(presentationController)
+        if #available(iOS 18.0, *), UIDevice.current.userInterfaceIdiom == .pad {
+            config.presentationContext = .window(windowLevel: .normal)
+        } else {
+            if let presentationController {
+                config.presentationContext = .viewController(presentationController)
+            }
```

Comment:
Combining the `else` with the enclosed `if` is a bit more readable:

```suggestion
        }
        else if let presentationController {
            config.presentationContext = .viewController(presentationController)
```